codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/testutil/TestHelpers.kt

```diff
@@ -41,6 +44,32 @@ val TestSymbolVisitorConfig = SymbolVisitorConfig(
     nullabilityCheckMode = NullableIndex.CheckMode.CLIENT_ZERO_VALUE_V1,
 )
 
+fun clientTestRustSettings(
```

Comment:
After #1762, whenever we move the unit tests from `codegen-client` to `codegen-core`, this function will stay in `codegen-client`'s unit tests and the one below will be moved to `codegen-core`.